futures-io/src/lib.rs

```diff
@@ -205,6 +208,10 @@ mod if_std {
         /// `Interrupted`.  Implementations must convert `WouldBlock` into
         /// `Poll::Pending` and either internally retry or convert
         /// `Interrupted` into another error kind.
+        ///
+        /// It only makes sense to do anything here if you actually buffer data. Otherwise,
```

Comment:
I'm not sure this paragraph is helpful beyond perhaps the first sentence.

Reply:
ok, I shortened it.